From 9db3188abd7218d406b873cc7c11aa003407cb6a Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Mon, 20 Nov 2017 15:41:08 +0800
Subject: [PATCH] ARM: dts: rk312x-android: fix for reserved memory

1. remove "linux,cma-default", which used by cma default region
2. use size=< > to define region size

Change-Id: Ie801b3ecebf7c8b4118700f83c4284c42337b0c2
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk312x-android.dtsi | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rk312x-android.dtsi b/arch/arm/boot/dts/rk312x-android.dtsi
index ade77b9080b2..4a2b5e6b1c1d 100644
--- a/arch/arm/boot/dts/rk312x-android.dtsi
+++ b/arch/arm/boot/dts/rk312x-android.dtsi
@@ -123,11 +123,10 @@
 		#size-cells = <1>;
 		ranges;
 
-		cma_region: region@6a000000 {
+		cma_region: cma-region {
 			compatible = "shared-dma-pool";
-			no-map;
-			reg = <0x6a000000 0x1000000>;
-			linux,cma-default;
+			reusable;
+			size = <0x1000000>;
 		};
 
 		ramoops_mem: ramoops@68000000 {
-- 
2.35.3

